pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

```diff
@@ -32,42 +31,23 @@ release "elasticsearch-kubeaddons" uninstalled
 
 ## Istio
 
-If Istio is installed on the cluster, the upgrade leaves the Helm Secret
-that belonged to the 1.8 Istio Addon in the `istio-system` namespace.
+If Istio is installed on the cluster, the upgrade leaves the Helm Secret that belonged to the 1.8 Istio addon in the `istio-system` namespace.
 
-**WARNING:** As long as this secret exists, `helm list -n istio-system`
-will incorrectly report the existence of a Helm release called "istio-kubeaddons".
-If you manipulate this "Helm release" using Helm, it may corrupt the Istio
-Platform Application in Kommander, potentially disrupting operation of
-Istio workloads.
+<p class="message--note"><strong>WARNING: </strong>As long as this secret exists, <code>helm list -n istio-system</code> will incorrectly report existence of an "istio-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Istio Platform Application in Kommander, which can disrupt operation of Istio workloads.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as this secret exists, <code>helm list -n istio-system</code> incorrectly reports the existence of an "istio-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Istio Platform Application in Kommander, which can disrupt operation of Istio workloads.</p>
```